### ISSUE TITLE
Admin doesn't handle skaled error

### ISSUE THREAD

**Issue by yatsunastya** [skalenetwork/skale-admin#1081]
**Describe the bug**
There is a problem in skaled https://github.com/skalenetwork/skaled/issues/1922. 
Admin doesn't handle this issue:
```
[2024-06-14 11:53:48,570 INFO][18489][sparse-gross-lesath][T_1] - core.schains.firewall.utils:42 - Creating rule controller for sparse-gross-lesath
[2024-06-14 11:53:48,577 ERROR][18489][sparse-gross-lesath][T_1] - core.schains.task:26 - Task sparse-gross-lesath-skaled failed with 'result'
Traceback (most recent call last):
  File "/usr/src/admin/core/schains/task.py", line 24, in run
    self.action()
  File "/usr/src/admin/core/schains/monitor/main.py", line 162, in run_skaled_pipeline
    status = skaled_checks.get_all(log=False, expose=True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/admin/core/schains/checks.py", line 126, in get_all
    if hasattr(self, name):
       ^^^^^^^^^^^^^^^^^^^
  File "/usr/src/admin/core/schains/checks.py", line 402, in blocks
    return CheckRes(check_endpoint_blocks(http_endpoint))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/admin/core/schains/rpc.py", line 52, in check_endpoint_blocks
    latest_schain_timestamp_hex = res_data['result']['timestamp']
                                  ~~~~~~~~^^^^^^^^^^
KeyError: 'result'
[2024-06-14 11:53:48,578 INFO][18489][sparse-gross-lesath][T_1] - core.schains.task:27 - Sleeping after task execution for 2
```
As a result it prevents starting skaled monitor. 

**Versions:** 
Admin: 2.7.0-beta.1
Skaled: 3.19.0-beta.1

**To Reproduce**
1. Run schain creation/ node rotation 
2. Check admin logs

**Expected behavior**
Admin restarts schain container.

**Comment by badrogger** [comment]
From my point of view it should be fixed in 2.4

**Comment by yatsunastya** [comment]
As the skaled issue was fixed, it's not possible to check it.
